Author: Moritz Muehlenhoff <jmm@debian.org>
Description:
 Adding missing format strings in order to build with hardening flags.

diff -Naurp traceroute.orig/libsupp/clif.c traceroute/libsupp/clif.c
--- traceroute.orig/libsupp/clif.c	2012-04-26 09:43:18.442365190 +0200
+++ traceroute/libsupp/clif.c	2012-05-18 12:40:19.282319731 +0200
@@ -792,9 +792,9 @@ static void box_output (int start, int l
 	if (l > 0) {
 	    memset (buf, ' ', l);
 	    buf[l] = '\0';
-	    fprintf (stderr, buf);
+	    fprintf (stderr, "%s", buf);
 	} else 
-	    fprintf (stderr, spacer);
+	    fprintf (stderr, "%s", spacer);
 
 
 	endp = buf + width;
@@ -833,7 +833,7 @@ static void box_output (int start, int l
 		    
 	    *p = '\0';
 	    fprintf (stderr, "%s", buf);
-	    fprintf (stderr, spacer);
+	    fprintf (stderr, "%s", spacer);
 		
 	    p++;
 	    for (s = buf; *p; *s++ = *p++) ;
